src/Cms/LicenseStatus.php

```diff
@@ -43,6 +43,22 @@ enum LicenseStatus: string
 	 */
 	case Missing = 'missing';
 
+	/**
+	 * Checks if the license can be saved when it
+	 * was entered in the activation dialog;
+	 * renewable licenses are accepted as well
+	 * to allow renewal from the Panel
+	 */
+	public function activatable(): bool
+	{
+		return match ($this) {
+			static::Active   => true,
+			static::Inactive => true,
+			static::Legacy   => true,
+			default          => false
+		};
+	}
```

Comment:
Just two small suggestions, not essential for me
- `canBeActivated` could be an easier to parse name
- I like to use match statements that produce the same result in union, so

```php
			static::Active,
			static::Inactive,
			static::Legacy   => true,
```

Reply:
I agree on the name, but it would be inconsistent with the existing `renewable` name. Can `LicenseStatus` be considered internal so that we can change both?

Also agree on the `match` statement. We can do the same in `renewable`/`canBeRenewed`.

Reply:
I think we can/should consider it as internal, maybe needs the tag added if not yet. I'm fine with changing both or leaving activatable... you are right that it should be consistent.

Reply:
How strict do we want to be with semver here? I'm also fine with both ways.

Reply:
I think in general these classes/enums should be off limit for developers and we should mark them as such. Nobody should fiddle with our license code.

Reply:
But this means that we should at least make this change in 4.1 and also introduce the internal tags with the PR, right?

Reply:
True. I think timing wise we could release 4.1 then very soon in the new year.

Reply:
> Nobody should fiddle with our license code.

Not even sure TBH. Especially the status could be used in agency-internal project management plugins or monitoring tools like Kirby Fleet Manager.

For me the `@internal` tag is more for code that we don't consider stable or don't see a reason to document.

IMO I'd keep the `LicenseStatus` enum public and also keep the method names to avoid a breaking change. If we are all fine with both names, I don't think we should break SemVer for no real reason.

Reply:
Fair enough

Reply:
Works for me as well.